### ISSUE TITLE
[Bug]: please update boost 1.81.0

### ISSUE THREAD

**Issue by Victor9578** [termux/termux-packages#14085]
### Problem description

As of Python 3.11 we get the following error upon import of the binding;

SystemError: type Boost.Python.enum has the Py_TPFLAGS_HAVE_GC flag but has no traverse function

Please update 1.81.0,they have commited this version

### What steps will reproduce the bug?

when use deluge,

will 

`SystemError: type Boost.Python.enum has the Py_TPFLAGS_HAVE_GC flag but has no traverse function`

### System information

termux-info:

```
termux:0.118
python-cryptography:lastest
python:3.11.X
boost:1.80.0
```

**Comment by xtkoba** [comment]
I will take the task of bumping Boost to 1.81.0 itself. But,

can't the issue with "deluge" be resolved without bumping Boost? Note that I don't even know the way to reproduce the issue, because we have no such package as `deluge`.

**Comment by xtkoba** [comment]
Note that it is not easy to bump Boost because it involves revdep rebuild. It is not uncommon for packages to fail to build with a newer version of Boost. For example see <a href="https://bugs.gentoo.org/show_bug.cgi?id=886577">the tracker for Gentoo Portage</a>. Possibly it could be much easier to fix the issue with "deluge" without bumping Boost.

**Comment by Victor9578** [comment]
> 



> I will take the task of bumping Boost to 1.81.0 itself. But,
> 
> can't the issue with "deluge" be resolved without bumping Boost? Note that I don't even know the way to reproduce the issue, because we have no such package as `deluge`.

**reproduce**
---
pkg i python-cryptography,then pip install deluge and run deluged , you will get this result.
Now i flash th e kernal,can use deluge that is running in docker .
Thanks for your maintenance

**Comment by xtkoba** [comment]
Should be fixed with `boost` version `1.80.0-1` which will soon become available.
